Some typos

ThomasLandauer opened the pull request.

ThomasLandauer committed:
Some typos

javiereguiluz commented on service_container/3.3-di-changes.rst:
```diff
@@ -433,7 +433,7 @@ In this case, you've created a class that implements ``EventSubscriberInterface`
 that you want this to be used as an event subscriber: more configuration is not needed.
 And the tags system is its own, Symfony-specific mechanism. And of course, you can
-always default ``autoconfigure`` to false in ``services.yml``, or disable it for a specific
+always set ``autoconfigure`` to ``false`` (=default) in ``services.yml``, or disable it for a specific
```
I'd remove the `(=default)` mention here because we already explain the default config before in this article.

HeahDude replied:
Yes this line changes should be reverted.

ThomasLandauer replied:
Sure, see https://github.com/symfony/symfony-docs/pull/8230/commits/5bc4c4eba1f36ea9232a2fef53e16ad0a66db08a

HeahDude replied:
I would keep "default" instead of "set" since it's meant to be under `_default` key.

ThomasLandauer replied:
"default" is not a verb you can use in this context.

ThomasLandauer committed:
Update 3.3-di-changes.rst

As suggested by https://github.com/symfony/symfony-docs/pull/8230#pullrequestreview-52460591

xabbuh commented:
Thank you @ThomasLandauer.
